### ISSUE TITLE
API for adding Services

### ISSUE THREAD

**Issue by JanBednarik** [kiwicom/the-zoo#133]
New Services can be added only trough web UI now. We have a request for adding Services using API.

**Comment by aexvir** [comment]
Yep, I discussed this with @adam-janis a couple weeks ago, and it would be used for example in the same way as the automatic dashboards are created. As it will be automatically called it will provide us with more accurate data about our services without requiring manual input.

The API should support CRUD operations
